src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -275,31 +272,40 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
             emit ProjectCollateralFeeAdded(projectFeeAmount);
         }
 
-        // Calculate mint amount based on upstream formula
-        uint issuanceMintAmount = _issueTokensFormulaWrapper(netDeposit);
-        totalIssuanceTokenMinted = issuanceMintAmount;
+        // Calculate token amount based on upstream formula
+        uint issuanceTokenAmount = _issueTokensFormulaWrapper(netDeposit);
+        totalIssuanceTokenMinted = issuanceTokenAmount;
 
         // Get net amount, protocol and project fee amounts. Currently there is no issuance project
         // fee enabled
-        (issuanceMintAmount, protocolFeeAmount, /* projectFeeAmount */ ) =
+        (issuanceTokenAmount, protocolFeeAmount, /* projectFeeAmount */ ) =
         _calculateNetAndSplitFees(
-            issuanceMintAmount, issuanceBuyFeePercentage, 0
+            issuanceTokenAmount, issuanceBuyFeePercentage, 0
         );
         // collect protocol fee on outgoing issuance token
         _processProtocolFeeViaMinting(issuanceTreasury, protocolFeeAmount);
 
-        // Revert when the mint amount is lower than minimum amount the user expects
-        if (issuanceMintAmount < _minAmountOut) {
+        // Revert when the token amount is lower than minimum amount the user expects
+        if (issuanceTokenAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Mint tokens to address
-        _mint(_receiver, issuanceMintAmount);
-        // Emit event
+
+        // Use virtual function to handle issuance tokens
+        _handleIssuanceTokenAfterBuy(_receiver, issuanceTokenAmount);
+
         emit TokensBought(
-            _receiver, _depositAmount, issuanceMintAmount, _msgSender()
+            _receiver, _depositAmount, issuanceTokenAmount, _msgSender()
         );
     }
 
+    /// @notice Virtual function to handle issuance tokens after a successful buy.
+    /// @param  _receiver The address that used the buy function.
```

Comment:
It's just the address receiving the tokens, not necessarily the address that called the buy function (`buyFor`).

Reply:
The address might not directly be handled. I rephrased it to  The address for which the issuance tokens will be handled.